Build/Laptop/EMail.cc

```diff
@@ -1916,7 +1917,7 @@ static void HandleIMPCharProfileResultsMessage(void)
 		// have to place players name into string for first record
 		if (i == 0)
 		{
-			wchar_t zTemp[512];
+			wchar_t zTemp[NAME_LENGTH];
```

Comment:
Oh, right. Isn't it too low now though? NAME_LENGTH is 30, zName 32 (if I read it correctly) and the format string contains an extra space. So it could cut the string too short now.

The other question is, why is doing the catting separately?